tests/cursor/cursorinterface-003.phpt

```diff
@@ -1,11 +0,0 @@
---TEST--
-MongoDB\Driver\CursorInterface does not extend Iterator
---FILE--
-<?php
-var_dump(in_array('Iterator', class_implements('MongoDB\Driver\CursorInterface')));
-?>
-===DONE===
-<?php exit(0); ?>
---EXPECT--
-bool(false)
```

Comment:
Noted that we already have a test that checks that `CursorInterface` implements `Iterator`.